tool/tctl/common/auth_command.go

```diff
@@ -88,7 +89,7 @@ func (a *AuthCommand) Initialize(app *kingpin.Application, config *service.Confi
 	a.authExport.Flag("keys", "if set, will print private keys").BoolVar(&a.exportPrivateKeys)
 	a.authExport.Flag("fingerprint", "filter authority by fingerprint").StringVar(&a.exportAuthorityFingerprint)
 	a.authExport.Flag("compat", "export cerfiticates compatible with specific version of Teleport").StringVar(&a.compatVersion)
-	a.authExport.Flag("type", "certificate type: 'user', 'host', 'tls-host' or 'tls-user'").StringVar(&a.authType)
+	a.authExport.Flag("type", "certificate type: 'user', 'host', 'tls-host', 'tls-user', 'tls-user-der' or 'windows'").StringVar(&a.authType)
```

Comment:
We should probably do 

```
Flag("type", "certificate type").Enum("user", "host", "tls-host", ...)
```

Or better yet:

```
var allowedCertificateTypes = []string{"user", "host", ...}
Flag("type", ..).Enum(allowedCertificateTypes...)
```

This means less error checking we have to do, and the available options automatically show up in the help message.